Client/Frontend/Browser/Tabs/Middleware/TabManagerMiddleware.swift

```diff
@@ -192,16 +192,8 @@ class TabManagerMiddleware {
         defaultTabManager.selectTab(tab)
     }
 
-    private func setTabManager(_ tabManager: TabManager, for windowUUID: WindowUUID) {
-        tabManagers[windowUUID] = tabManager
-    }
-
-    private func removeTabManager(_ tabManager: TabManager, for windowUUID: WindowUUID) {
-        tabManagers.removeValue(forKey: windowUUID)
-    }
-
     private var defaultTabManager: TabManager {
         // TODO: [FXIOS-7863] Temporary. WIP for Redux + iPad Multi-window.
-        return tabManagers[.defaultSingleWindowUUID]!
+        return windowManager.tabManager(for: windowManager.activeWindow)
```

Comment:
cc @yoanarios I think this change should hopefully simplify things for Redux middlewares. They no longer need to keep track of the tab managers themselves (they shouldn't really have been doing that anyway, that responsibility should be separated out, so I've moved it into the new `WindowManager` class).